website/src/components/Homepage/Features/FeatureList.js

```diff
@@ -4,31 +4,37 @@ const FeatureList = [
   {
     title: 'Cross Platform',
     Svg: require('@site/static/img/homepage/cross-platform-light.svg').default, // taken from: https://uxwing.com/magic-icon/
+    image: require('@site/static/img/homepage/cross-platform-light.png').default, // taken from: https://uxwing.com/magic-icon/
```

Comment:
Let's rename this field to `png` instead of `image`, since SVG is another type of image format